src/views/TrialDescriptionPage/TrialDescriptionPage.jsx

```diff
@@ -35,9 +35,11 @@ const TrialDescriptionPage = () => {
 	const parsed = queryString.parse(qs);
 	const currId = parsed.id;
 
-	const trialTitle = useSelector((store) => store.cache.currentTrialTitle);
+	// Note: Location state only exists if navigating.
+	// If the trial description is directly navigated to this will be empty
+	// on first render
+	const trialTitle = location.state ? location.state.result.brief_title : '';
```

Comment:
Does the title show up when pasting in the url?

Reply:
we have test that covers direct navigation to trial https://github.com/NCIOCPL/clinical-trials-search-app/blob/develop/cypress/integration/TrialDescriptionPage/GeneralPageComponents.feature#L39
I didn't test locally but I've been trusting Cypress

Reply:
Ah, finally found its use. Couldn't earlier.